src/index.js

```diff
@@ -686,20 +686,17 @@ class NearProvider {
                 const remove = utils.remove0x(value)
                 val = new BN(remove, 16)
             }
-            // value === undefined
-            //     ? new BN(0)
-            //     : new BN(utils.remove0x(value), 16);
 
-            // TODO: differentiate simple sends
-            const functionCallData = {
-                contractId: this.evm_contract,
-                methodName: 'deploy_code',
-                args: { 'bytecode': utils.remove0x(data) },
-                gas: GAS_AMOUNT,
-                amount: val
-            }
-
-            if (to === undefined) {
+            if (to != "0x0000000000000000000000000000000000000000" && to == txObj.from) {
```

Comment:
we need to `add near`  before we send funds to another evm address. Could this be done in one transaction?